src/Makefile.am

```diff
@@ -255,6 +262,11 @@ obj/build.h: FORCE
 	  "$(abs_top_srcdir)"
 libbitcoin_util_a-clientversion.$(OBJEXT): obj/build.h
 
+# libbitcoin_rustybridge should not need headers other than Core's internal ones.
+libbitcoin_rustybridge_a_CPPFLAGS = $(AM_CPPFLAGS) -I$(builddir) $(LEVELDB_CPPFLAGS)
+libbitcoin_rustybridge_a_CXXFLAGS = $(AM_CXXFLAGS) $(PIE_FLAGS)
+libbitcoin_rustybridge_a_SOURCES =
+libbitcoin_rustybridge_a_SOURCES += rusty/src/cpp_bridge.cpp
```

Comment:
I probably don't know Makefile syntax minutiae enough, but why not just:
```
libbitcoin_rustybridge_a_SOURCES = rusty/src/cpp_bridge.cpp
```

Reply:
I think its just historical, fixed.

Reply:
Fwiw, I've done this as a convention so that future diffs are 100% straightforward when adding/removing files and keeping them in alphabetical order. I don't really care much either way though.